event_ticket_maxlimit/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from . import models
```

Comment:
```suggestion
from . import models
```
no new line at end of file :eyes: